src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -6,20 +6,20 @@
     <ProjectGuid>{4C2655DB-BD9E-4C86-83A6-744ECDDBDF29}</ProjectGuid>
     <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
     <UseOpenKey Condition="'$(UseOpenKey)'==''">true</UseOpenKey>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETCoreApp,Version=v1.1</NuGetTargetMoniker>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == ''">true</IsPartialFacadeAssembly>
+    <RunApiCompat Condition="'$(TargetGroup)' != ''">false</RunApiCompat> 
```

Comment:
While I realize you disable this because of some limitations in our targets I would suggest you manually run it at least once to ensure your ref and implementation match.